tox.ini

```diff
@@ -8,6 +8,9 @@ skip_missing_interpreters = true
 
 [testenv:docs]
 skip_install = false
+install_command =
```

Comment:
Adding `Program` back as importable from the `core` package is right, but do we want the hosted docs to reflect the status of `main` branch? I thought we decided a while back it was better for docs to reflect the status of the latest release?

Reply:
we just had an interesting problem because of this. Basically with older docs approach it was impossible to change documentation without a Pypi release. Because sphinx was building docs from sourcecode that is in PyPI.

Reply:
Yes, but users will generally be installed from `PyPI`, and when they visit our docs, we want them to reflect that, right? `main` branch is more likely to change day-to-day, and I feel like users will be more likely to be working off a release.

The only reason there was an error was because the `Program` was removed as importable but remained in the docstring. It shouldve been cleaned up in both or remained in both.

It's up to you, but I do think building docs from PyPI is better than main.

Reply:
on top of that CI was doing basically nothing with old approach and was checking not sourcecode of repository, but what was on Pypi 😓

Reply:
Another option might be to run CI on the `main` branch docs build, but continue building our `qiskit.org` docs from the release. If we did this, it would have caught the error caused by removing `Program` in one place but not the other. Just a thought :)